.github/workflows/edr-npm-release.yml

```diff
@@ -0,0 +1,329 @@
+name: EDR NPM release
```

Comment:
This file has a lot of code duplication and thus is prone to error. Another downside of the workflow being encoded in the GitHub CI instead of a Rust CLI app is that it cannot be reproduced locally.

With the knowledge you've gained from doing this, can you easily avoid duplication and configure it in a Rust CLI app?

Reply:
Yes I agree that this needs to be improved. It's not trivial, but definitely doable. I created https://github.com/NomicFoundation/edr/issues/198 as a follow up.